website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -1,24 +1,21 @@
 ## How to use this component
 
 Super Select is a custom select-like component aiming to overcome some limitations of the HTML `<select>` element.
-It is based on the popular Ember [PowerSelect](https://ember-power-select.com/) add-on.
 
-We provide two main components with similar APIs, `SuperSelect::Single` and `SuperSelect::Multiple`.
+It's a light wrapper for Ember [PowerSelect](https://ember-power-select.com/), with some specific accessibility and styling choices made to best fit our design system.
```

Comment:
I don't know if I would characterize it as just a "light" wrapper. Maybe remove that word?

Reply:
How about, "It’s primarily a wrapper for..."